File: Day 5/loops_v2.py
num_of_tests = input()
ans = ''
for test in range(num_of_tests):
    str_in = raw_input().split(' ')
    a = int(str_in[0])
    b = int(str_in[1])
    N = int(str_in[2])
    current_sum = a
    for i in range(N):
        current_sum += 2 ** i * b
        ans += str(current_sum) + ' '
    ans += '\n'

print ans[:-1]

File: Day 5/loops_v2.java
import java.util.*;

public class Solution {
    public static void main(String[] args) {
        String ans = "";
        Scanner scIn = new Scanner(System.in);
        int numOfTests = scIn.nextInt();
        for (int i = 0; i < numOfTests; ++i) {
            int a = scIn.nextInt();
            int b = scIn.nextInt();
            int N = scIn.nextInt();
            int currentSum = a;
            for (int j = 0; j < N; ++j) {
                currentSum += (int)Math.pow(2, j) * b;
                ans += currentSum + " ";
            }
            ans += "\n";
        }
        System.out.print(ans);
    }
}